src/app/_components/DescriptionText.tsx

```diff
@@ -1,18 +1,25 @@
 type DescriptionProps = {
   children: DescriptionTextType
+  variant?: 'bold'
 }
 
 export type DescriptionTextType = string | string[]
 
-export function DescriptionText({ children }: DescriptionProps) {
+export function DescriptionText({ children, variant }: DescriptionProps) {
+  const fontStyles = {
+    bold: 'text-xl font-bold',
+  }
+
   if (Array.isArray(children)) {
     return (
       <>
         {children.map((item, index) => (
-          <p key={index}>{item}</p>
+          <p key={index} className={variant && fontStyles[variant]}>
+            {item}
+          </p>
         ))}
       </>
     )
   }
-  return <p>{children}</p>
+  return <p className={variant && fontStyles[variant]}>{children}</p>
 }
```

Comment:
I think this may become irrelevant - Since I don't think we need a description in `BlogPostHeader`

Reply:
So the [first paragraph below the image](https://www.figma.com/file/0pBfQs5tKI6bx5ypPIfVRA/FIL.org-V4-Prototypes?type=design&node-id=2903-34786&mode=design&t=LamjUrVlkFq5LVk3-4) is a part of the text?

Reply:
Yes - I think this was a bit unfortunate because these styles were copy/pasted - It's probably bold because of inline HTML in the markdown content - something we need to address at some point.